File: 백준/Bronze/9498. 시험 성적/시험 성적.js
const input = +require("fs").readFileSync(process.platform === "linux" ? "/dev/stdin" : "알고리즘/example.txt").toString().trim()

function calc_grade(score){
    if (score <= 100 && score >= 90){
        console.log('A');
    }
    else if (score <= 89 && score >= 80){
        console.log('B');
    }
    else if (score <= 79 && score >= 70){
        console.log('C');
    }
    else if (score <= 69 && score >= 60){
        console.log('D')
    }
    else {
        console.log('F');
    }
}

calc_grade(input);

File: 백준/Bronze/9498. 시험 성적/시험 성적.py
score = int(input())

if score <= 100 and score >= 90:
    grade = 'A'
elif score < 90 and score >= 80:
    grade = 'B'
elif score < 80 and score >= 70:
    grade = 'C'
elif score < 70 and score >= 60:
    grade = 'D'
else:
    grade = 'F'

print(grade)